packages/medusa-interfaces/src/base-model.js

```diff
@@ -67,8 +67,16 @@ class BaseModel {
    * @return {Array<mongoose.Document>} the retreived mongoose documents or
    * an empty array
    */
-  find(query, options) {
-    return this.mongooseModel_.find(query, options).lean()
+  find(query, options, offset, limit) {
+    return this.mongooseModel_
+      .find(query, options)
+      .skip(offset)
+      .limit(limit)
```

Comment:
Do these work when offset and limit are undefined?

Reply:
Tested with `list-customers`, that does not have these yet. Still works :)